testsuite/fixtures/crates/hello_1.0.0_8cac0afd/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is not required; I think we should avoid all unecessary files in testcase material, to simplify as much as possible the test environment. What do you think?

Reply:
OK. I just copied the crates as-is, but I see your point.